plugins/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3Repository.java

```diff
@@ -51,6 +54,8 @@
  * </dl>
  */
 class S3Repository extends BlobStoreRepository {
+    private static final Logger logger = LogManager.getLogger(S3Repository.class);
```

Comment:
this is a nit but typically static finals should be all uppercase (`LOGGER` and `DEPRECATION_LOGGER`)

Reply:
It would be great to have a standard across the team for this; I've been asked to use all uppercase before so I adopted it, but I get the point of the google style too. Anyway, this shouldn't block your change